app/components/pages/Dashboard/Dashboard.js

```diff
@@ -4,12 +4,23 @@ import { Link } from 'react-router-dom'
 import { Box } from 'grid-styled'
 import { Button } from '@pubsweet/ui'
 
+import media from '../../global/layout/media'
 import { FormH2, FormH3 } from '../../ui/atoms/FormHeadings'
 import Paragraph from '../../ui/atoms/Paragraph'
 import ManuscriptStatus from '../../ui/atoms/ManuscriptStatus'
+import StickyFooter from '../../ui/atoms/StickyFooter'
 
 /* Temporary dashboard view pending receipt of designs */
 
+const StyledBox = styled(Box)`
+  display: none;
+  ${media.mobile`display:block;`};
+`
```

Comment:
The name `StyledBox` is quite meaningless. Maybe `HideFromMobile`?

Reply:
Done :+1: